### PR TITLE
Fix table of contents list

### PR DESCRIPTION
**Pull request by chrishelgert**
Looks like mixing details with Markdown content does not work correctly.

broken-content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,24 +44,38 @@ Fly is extremely extensible, so _anything_ can be a task. Our core system will a
 <h2>Table of Contents</h2>
 <details>
 <summary>Table of Contents</summary>
-- [Features](#features)
```
Comment (hzlmn) [inline README.md]: I am pretty sure it worked earlier, seems like some updates to GitHub's markdown parsing. Can you just add new line here. It is not convenient to use lots of tags inside markdown.

**Review by hzlmn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chrishelgert) [reply README.md]: sorry, what do you mean with `Can you just add new line here`?
just write each link (markown or tag) in a new line without a list?

Comment (hzlmn) [reply README.md]: No, I mean just add new line on line `47` without rewriting structure to tags and it will fix the problem.

Comment (jorgebucaran) [reply README.md]: @hzlmn I wonder if this has anything to do with GitHub's wikis breaking down.

Comment (jorgebucaran) [reply README.md]: See: https://github.com/flyjs/fly/wiki/Quickstart

Colors are gone.

Comment (hzlmn) [reply README.md]: @jbucaran Yep, seems like GitHub [updated](https://github.com/blog/2333-a-formal-spec-for-github-flavored-markdown) its `markdown` processing. So need to check spec and update all repos.

Comment (chrishelgert) [reply README.md]: moved it back to markdown and added the empty line. is really the easier fix

**Comment by lukeed** [comment]
It did work. And it was beautiful 😫 The recent markdown changes killed it.

**Review by hzlmn — APPROVED** [review]

**Comment by lukeed** [comment]
Thanks very much for fixing it! 🎉 😃

**Comment by chrishelgert** [comment]
my pleasure 😄

**Pull request merged**